src/elements/common/item/__tests__/IconCell.test.js

```diff
@@ -19,6 +19,32 @@ describe('elements/common/item/IconCell', () => {
             expect(wrapper.prop('title')).toBe('File');
         });
 
+        test('should render archive icon', () => {
```

Comment:
Please don't create any new tests using Enzyme. Reimplement them using React Testing Library. It would be great if you converted some other tests along the way.

Reply:
Done